Tools/PRTest.ps1

```diff
@@ -16,38 +15,44 @@ Param(
     [switch] $Clean
 )
 
-$PullRequest = $PullRequest.TrimStart('#')
+# Virtual Terminal
+filter Initialize-VirtualTerminalSequence {
+    if ($script:vtSupported) {
+        return "$([char]0x001B)[${_}m"
```

Comment:
Can we add some references or explain more in comments as to where this sequence is coming from?